lib/ex_doc/formatter/html.ex

```diff
@@ -72,22 +72,24 @@ defmodule ExDoc.Formatter.HTML do
       extras: extra_paths(config),
       skip_undefined_reference_warnings_on: config.skip_undefined_reference_warnings_on,
       skip_code_autolink_to: config.skip_code_autolink_to,
+      force_module_prefix: config.force_module_prefix,
```

Comment:
What is the use case for this one? :)

Reply:
Either to allow elixir fork to forced their projects to use the `m:` prefix (from the other threads I gathered that possibly @wojtekmach wanted this?). Also, it is for people that are used to writing docs in Elixir and want to keep the same ergonomics in Erlang docs.

It was easy to add, so I added it. I'm fine with removing it if you think it makes things more complicated than they need to be.

Reply:
do you mean that Elixir folks could configure their project to require writing `` `m:String` `` because otherwise it won't autolink? Yeah let's skip this for now.

Reply:
btw, In Elixir project I noticed `` `m:EarmarkParser` `` creates `<a href=...><code>m:EarmarkParser</code></a>`, that is, it does _not_ remove the `m:` prefix in the renderer text. I also noticed `m::lists` does _not_ autolink. That being said, I think you don't explicitly document `m:` working for Elixir so I believe it's fine to be rough around the edges for now.

Reply:
I'll remove the config and fix the `m:` prefix removal in elixir.

Regaring `m::lists`, it should work but it only works as the link target (i.e. a `:custom_link`). `:lists` has as far as I can tell never worked as a ":regular_link".

Reply:
as I hit enter I realized that the `m:` should not be used in `:custom_links`. So I'll leave that to you to figure out if you want to allow `m::lists` in regular links when `:lists` is not allowed.

Reply:
agreed, there's no need for `m:` in custom links. (Come to think of it, it's less about whether it's regular vs custom link it's all about whether we're doing relaxed vs strict reference parsing. Should better express that in code one of those days.)

I think `m::lists` would be a nice addition for Elixir projects but it's not high priority at the moment IMHO. WDYT @josevalim?

Reply:
Regarding this option, let's remove it. I think it is fine for Erlang to be more explicit here (`m:...`) while Elixir allows both, especially since modules in Elixir have a different casing than regular atoms. :) If someone reaaaaaaally wants it, they can send a PR later :D

Reply:
`` `m:Enum` `` works as it should now in Elixir. But `` `m::lists` `` does not create a link. I'll leave things like this and you can update the elixir parts if you want to have `` `m::lists` ``.

I've removed the option to enforce `m:`.